docs/source/code-generation/codegen-configuration.mdx

```diff
@@ -426,13 +426,8 @@ The top-level properties are:
   "deprecatedEnumCases": "include",
   "schemaDocumentation": "include",
   "selectionSetInitializers" : {
-    "operations": false,
-    "namedFragments": false,
-    "localCacheMutations" : true,
-    "definitionsNamed": [
```

Comment:
The `definitionsNamed` should still be available. I see you are now setting `operations` and `namedFragments` to `true` so it's not needed, but I had them `false` so that it was relevant to show the example of the `definitionsNamed` options as well.